lib/components/position_component.dart

```diff
@@ -25,6 +24,10 @@ abstract class PositionComponent extends BaseComponent {
   /// The position of this component on the screen (relative to the anchor).
   Vector2 position = Vector2.zero();
 
+  /// If the component has a [PositionComponent] as a parent it will be
+  /// added here
+  PositionComponent positionParent;
```

Comment:
This should be private to avoid people messing around with this field?

Reply:
True, do we want a getter for it, or should we skip exposing it for now?

Reply:
Initially, this seems to internal for me to be exposed, but I wouldn't oppose a getter as well.

Reply:
Yeah, let's keep it internal until the day someone finds a usecase for it.